incubator/hnc/internal/validators/anchor_test.go

```diff
@@ -62,14 +62,15 @@ func TestAllowCascadingDeleteSubnamespaces(t *testing.T) {
 		acd  string
 		pnm  string
 		cnm  string
+		stt  api.SubnamespaceAnchorState // anchor state, "ok" by default
 		fail bool
 	}{
 		{name: "set in parent", acd: "c", pnm: "c", cnm: "d"},
 		{name: "set in non-leaf", acd: "d", pnm: "c", cnm: "d"},
 		{name: "set in ancestor that is not the first full namespace", acd: "a", pnm: "c", cnm: "d"},
 		{name: "unset in leaf", pnm: "d", cnm: "e"},
 		{name: "unset in non-leaf", pnm: "c", cnm: "d", fail: true},
-		{name: "unset in non-leaf but bad anchor", pnm: "b", cnm: "d"},
```

Comment:
I think we still need to keep this one. This is the case for deleting a bad (nonsense) anchor. Maybe add `stt:api.Conflict` to it and keep it?

Reply:
Done